doc/shared/sundials/Install.rst

```diff
@@ -523,9 +523,11 @@ illustration only.
 
 .. cmakeoption:: CMAKE_CUDA_ARCHITECTURES
 
-   Specifies the CUDA architecture to compile for.
+   Specifies the CUDA architecture to compile for i.e., ``60`` for Pascal,
+   ``70`` Volta, ``80`` for Ampere, ``90`` for Hopper, etc.
```

Comment:
Can we provide a URL or something to reference where users can look to find these numbers for a given GPU?